Detectors/GlobalTrackingWorkflow/src/tfidinfo-writer-workflow.cxx

```diff
@@ -19,7 +19,7 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
 {
   // option allowing to set parameters
   std::vector<ConfigParamSpec> options{
-    ConfigParamSpec{"dataspec", VariantType::String, "tfidinfo:FLP/DISTSUBTIMEFRAME/0", {"spec from which the TFIDInfo will be extracted"}},
+    ConfigParamSpec{"dataspec", VariantType::String, "tfidinfo:FLP/DISTSUBTIMEFRAME/0xccdb", {"spec from which the TFIDInfo will be extracted"}},
```

Comment:
This does not solve the problem for running `$O2DPG_ROOT/DATA/production/configurations/2021/OCT/apass3/async_pass.sh` for me.

I am getting:
`[FATAL] error while setting up workflow in o2-tfidinfo-writer-workflow: Parse error: Expected a number`

Reply:
@chiarazampolli @shahor02

Reply:
Needs to be decimal. I will add support for parsing hex...

Reply:
#8298 should add support for 0xccdb (assuming Ruben is using DataDescriptorQueryBuilder::parse) is used.